v2/crd/kubeflow.org_mpijobs.yaml

```diff
@@ -0,0 +1,6908 @@
+
```

Comment:
Shouldn't this replace the old file?

Reply:
not yet, I want to also patch the commons api so the generated CRD has defaults as it has right now

Reply:
For now, this patch just demonstrates that the CRD can be generated, and shows how it looks

Reply:
gotcha. Let's get the changes in the common api first.